Carousel text

fjp2cornell opened the pull request:
Aaron, do you recall why article was given eq states of 100, 275 and 350? I couldn't find where they were being referenced in the styles. Changing them to match our other eq states of 320, 550, 768 and 1280 helps resize the link in the carousel on the home page.

a-fro commented on docroot/sites/all/themes/ilr_theme/template.php:
```diff
@@ -143,7 +143,7 @@ function ilr_theme_preprocess_node(&$variables) {
       );
     }
     // See note above about hard-coded eq-pts
```
@fjp2cornell the comment here says to see the note above, which will point you in the right direction. The link is in the docs for this function, ie `@see generate_event_item_markup() for hard-coded data-eq-pts`.

a-fro commented on docroot/sites/all/themes/ilr_theme/js/ilr_theme.js:
```diff
@@ -404,4 +404,22 @@
     }
   };
 
```
Is there a reason we need javascript for this? Could we use the :hover pseudo-selector, ie `.node-promo:hover a`?

fjp2cornell replied:
Probably. I've been doing a lot of js recently that I probably overlooked the obvious.

fjp2cornell commented:
@a-fro Removed the js and replaced with the pseudo-selector. Refactored styles to put carousel (highlighted), styles into separate file. The & and context mixin were not working properly, I think because of nested extends.

a-fro commented:
@fjp2cornell take a look at http://branch-carousel-text.ilr.featuretest.org/centers-institutes/international-programs/visiting-fellows/lisa-schulte. Is that behavior related to this pr? If so, any other unintended consequences?

fjp2cornell commented:
@a-fro yes, I forgot those content types were using the highlighted id. I'll have to fix it. Thanks for checking

a-fro commented:
Hey John, thanks for working on this. Take a look at the [refactored branch](https://github.com/ilrWebServices/ilr-website/compare/carousel-text-refactor) and let me know your thoughts.
